common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -25,6 +25,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/temporalio/temporal/.gen/proto/persistenceblobs"
```

Comment:
Let's group all imports in 3 blocks. I did it for the most cases in my other PR.